app/src/main/java/com/example/trempel/PdpFragment.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.trempel
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+
+class PdpFragment : Fragment() {
+    lateinit var productDescription: TextView
```

Comment:
check if we should add line break after the brace https://kotlinlang.org/docs/coding-conventions.html#class-headers